visualize-admin/visualization-tool
# In place interactive filters

This moves the checkbox for "interactive" from the dedicated "interactive filters" section to the data filters and segment settings. The date filters changes will be in another PR.

## How to test

- Go to any dataset
- "Interactive" switches should be inside the "Filters" section on the left
- "Interactive" switch is also available for segments
